docs/standard/assembly/find-fully-qualified-name.md

```diff
@@ -1,11 +1,11 @@
 ---
 title: "How to: Find an assembly's fully qualified name"
-description: This article shows you how to get the fully qualified name of a .NET Framework assembly or .NET Core assembly.
+description: This article shows you how to get the fully qualified name of a .NET assembly.
 ms.date: "08/20/2019"
 helpviewer_keywords:
-  - "names [.NET Framework], fully qualified type names"
-  - "names [.NET Framework], assemblies"
-  - "assemblies [.NET Framework], names"
+  - "names [.NET , fully qualified type names"
+  - "names [.NET], assemblies"
+  - "assemblies [.NET], names"
```

Comment:
(this goes for the rest of these changes in other files) Would it be useful to duplicate these, one for each? If someone is in a .NET Framework app would they lose this context in the helpviewer/f1 finder? Unsure.

Also, there is a missing `]` in the first one.

```suggestion
  - "names [.NET] , fully qualified type names"
  - "names [.NET], assemblies"
  - "assemblies [.NET], names"
  - "names [.NET Framework], fully qualified type names"
  - "names [.NET Framework], assemblies"
  - "assemblies [.NET Framework], names"
```

Reply:
My take on this is it's not necessary to duplicate them. I'm not sure how many people actually use offline help, and as far as I'm aware, no one is really adding helpviewer keywords to new topics. So I feel this metadata has sort of fallen by the wayside. Plus, I think .NET vs .NET Framework is close enough that someone looking for .NET Framework help would still click the link.